src/PresentationalComponents/InfoBox/InfoBox.js

```diff
@@ -1,11 +1,13 @@
 import {
-    Bullseye,
-    Split,
-    SplitItem,
     Text,
     TextContent,
     TextVariants
-} from '@patternfly/react-core';
+} from '@patternfly/react-core/dist/js/components/Text/';
+import { Bullseye } from '@patternfly/react-core/dist/js/layouts/Bullseye/';
```

Comment:
@jiridostal does the `/ ` in the end really need it?

Reply:
Yes I think so as it's path that points to "....../components/Text/index.js"